domain/schema/testing/schema.go

```diff
@@ -23,6 +23,6 @@ func (s *SchemaApplier) Apply(c *gc.C, ctx context.Context, runner database.TxnR
 	})
 	changeSet, err := s.schema.Ensure(ctx, runner)
 	c.Assert(err, gc.IsNil)
-	c.Check(changeSet.Current, gc.Equals, 0)
+	// c.Check(changeSet.Current, gc.Equals, 0)
```

Comment:
Reinstate me.

Reply:
I've looked at this a bit more closely and I am satisfied that this check can be removed.  Specifically this check assumes we only fetch a DB at most once in a test run and the ServiceFactory pattern allows this to be multiple times per test possibly.

On to that this logic is tested very well in the schema tests and I don't think there is anything being asserted here.